### DISCUSSION TITLE
请问这个支持english version么

### DISCUSSION THREAD

**Discussion by stephenli290** [hhyo/Archery#1328] [Q&A]
请问这个支持English verison么？

**Comment by LeoQuote** [comment]
有国际化模块, 但是英语的要做的还是挺多的....现在没有英文版

**Reply by stephenli290** [reply]
这个英文版有计划支持么？

**Reply by stephenli290** [reply]
或者有推荐的类似英文版本开源sql 审核平台么？

**Reply by LeoQuote** [reply]
你看看 [shift ](https://github.com/square/shift) 英文版不是我们支持不支持, 这个是开源软件, 如果你有兴趣有能力, 就过来提 pr, 我们肯定是欢迎的.

**Reply by nick2wang** [reply]
主要不清楚有英文需求的有多少，太少了的话暂时就没这个动力去做

**Reply by stephenli290** [reply]
嗯，我们之前主要国内员工，用这个还是挺好用的。当前单位好多老外，国外开源貌似也没有特别成熟，跟这个类似的。我们当前主要是使用方，感觉这个工具挺好用的

**Reply by LeoQuote** [reply]
django 有国际化模块的, 做起来应该不会太难, 对我们来说就是 roe 太低的问题, 欢迎 pr

**Reply by stephenli290** [reply]
明白，谢谢

**Reply by maxzhang2017** [reply]
是否可以支持下英文，我们也有很多外籍的同事

**Reply by redlert** [reply]
我们这也有英文版的需求

**Comment by redlert** [comment]
> 模块, 但是英语的要做的还是挺多的....现在没有英文版

如果您实现太忙, 可以尝试把任务分发下来, 我们各自领取一些, 一起翻译掉.

**Reply by LeoQuote** [reply]
翻译是一方面，还有一个重要的工作是需要更改相关的引用代码，需要用django 国际化和本地化模块，欢迎pr

**Reply by LeoQuote** [reply]
这也并不是谁来分发任务的问题，如果你愿意，你可以搭设基本框架，你来分发任务，只要没有breaking change，代码风格和质量没有大问题，接受PR会很容易
